### PR TITLE
chore: configuring sequelize with nest kit

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Type of change

- [ ] Documentation change
- [ ] Bug fix
- [x] feature

## Summary of change

Tested this out with Postman
Screenshot 2023-03-30 at 10 41 35 AM

## Checklist

- [ ] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [ ] This starter kit has been approved by the maintainers
- [ ] Changes for this new starter kit are being pushed to an integration branch instead of main
- [ ] All dependencies are version locked
- [ ] This fix resolves #
- [ ] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/nestjs-bullmq-postgresql/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		"start:dev": "nest start --watch",
 		"start:debug": "nest start --debug --watch",
 		"start:prod": "node dist/main",
+		"start:all": "yarn infrastructure:start && yarn start:dev",
```
Comment (ktrz-sre-is) [inline starters/nestjs-bullmq-postgresql/package.json:15]: Do we have a convention for naming the scripts like that? Is `start:all` following that?

**Review by ktrz-sre-is — APPROVED** [review]
Grat work! That looks great! Just added one question

File: starters/nestjs-bullmq-postgresql/package.json (hunk 1 above)
Comment (ktrz) [inline starters/nestjs-bullmq-postgresql/package.json:15]: Do we have a naming convention for that kind of scripts? Does `start:all` follow that?

**Review by ktrz — APPROVED** [review]
Great work! Added a small question but LGTM

**Pull request merged**